legacy/PimExportHelper.js

```diff
@@ -43,15 +43,21 @@ class PimExportHelper {
    * @param {string} fieldApi
    */
   getValue(object, fieldApi) {
+    if (!object || !fieldApi) {
+      return null
+    }
     let fields = fieldApi.split('.');
     let queryResult = object;
-    fields.forEach(field => {
+    for (const field of fields) {
```

Comment:
why are you moving away from the forEach?

Reply:
So i can use break to prevent null pointer

Reply:
and you can just check if it's there and then process. 
```
fields.forEach((field) => {
  if (queryResult) {
    if (field.endsWith('__c') || field.endsWith('__r')) {
        queryResult = queryResult[this.namespace(field)];
      } else {
        queryResult = queryResult.field;
      }
  }
});
```
But what ever is clever.